unit_tests/tests/test_sensors.cpp

```diff
@@ -54,32 +54,3 @@ TEST(sensors, testTpsRateOfChange) {
 //	saveTpsState(3 * CH_FREQUENCY, 75);
 //	assertEquals(25, getTpsRateOfChange());
 }
-
-TEST(sensors, Thermistor1) {
-
-	ThermistorMath tm;
-	thermistor_conf_s tc = {32, 75, 120, 9500, 2100, 1000, 0};
-	tm.setConfig(&tc);
-
-	float t = tm.getKelvinTemperatureByResistance(2100);
-	ASSERT_FLOAT_EQ(75 + KELV, t);
-
-	ASSERT_NEAR(-0.003, tm.s_h_a, EPS4D);
```

Comment:
why is this unit test removed?

Reply:
it moved to `tests/sensor/therm_func.cpp`, since `ThermistorMath` no longer exists